src/Altinn.Platform/Altinn.Platform.Events/Tests/Data/Roles/user_1/party_1000/roles.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
How many of these roles are actually in use in the tests? I would like to see that we don't duplicate testdata that we do not need at the moment. Our repository is large enough as is.

Reply:
Removed roles for unused users